File: server/DataCore/XUser.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace inv19.models { 
	/* XUser -  Оператор */ 

 public class  xUserInfo { // Оператор
	 public System.Guid  xUserInfoId{ get; set; } // Идентификатор (первичный ключ)
	[Required]
	public string  family{ get; set; } // Фамилия
	[Required]
	public string  name{ get; set; } // Имя
	public string  partonymic{ get; set; } // Отчество
	public string  email{ get; set; } // e-mail
	public string  phone{ get; set; } // Телефон
	[Required]
	public string  login{ get; set; } // Имя для входа
	public enum_YesNo  islocked{ get; set; } // Заблокирован
 }
}

File: client/src/app/XUser.ts
import { enums } from './enums';

export namespace XUser { 
	/* XUser -  Оператор */ 

 export interface   xUserInfo { // Оператор
	xUserInfoId:string; // Primary key field
	family:string; // Фамилия
	name:string; // Имя
	partonymic:string; // Отчество
	email:string; // e-mail
	phone:string; // Телефон
	login:string; // Имя для входа
	 islocked:enums.enum_YesNo; // Заблокирован
	 islocked_name :string; // enum to text for Заблокирован
 }
}